src/libexpr/primops.cc

```diff
@@ -56,6 +56,9 @@ void EvalState::realiseContext(const PathSet & context)
             throw InvalidPathError(store->printStorePath(ctx));
         if (!decoded.second.empty() && ctx.isDerivation()) {
             drvs.push_back(StorePathWithOutputs{ctx.clone(), {decoded.second}});
+            if (evalSettings.logAllIFD) {
+              printInfo(format("%1% depends on %2% via %3%") % pos % decoded.first % reason);
```

Comment:
maybe something a bit more explicit, mentioning "importing from derivation"?

Reply:
Also in the help text please.